packages/network-controller/src/NetworkController.ts

```diff
@@ -377,7 +391,9 @@ export class NetworkController extends BaseControllerV2<
 
   #safelyStopProvider(provider: Provider | undefined) {
     setTimeout(() => {
-      provider?.stop();
+      if (provider) {
+        provider?.stop();
+      }
```

Comment:
Isn't this functionally the same?

Reply:
Good point; replaced in https://github.com/MetaMask/core/pull/1266/commits/8ee4d7ce22be73bd05bc8f9045d747bbe629d2d8